src/CMakeLists.txt

```diff
@@ -94,14 +99,20 @@ if(${PLATFORM} MATCHES "PLATFORM_DESKTOP")
     foreach(L ${LIBS_PRIVATE})
       set(PKG_CONFIG_LIBS_PRIVATE "${PKG_CONFIG_LIBS_PRIVATE} -l${L}")
     endforeach(L)
+  endif()
 
-  elseif(APPLE)
+  if(APPLE)
     # TODO extract framework location and name from ${LIBS_PRIVATE}
     # and specify them as -F and -framework instead of hardcoding
     foreach(F OpenGL OpenAL Cocoa)
       set(PKG_CONFIG_LIBS_PRIVATE "${PKG_CONFIG_LIBS_PRIVATE} -framework ${F}")
     endforeach(F)
+  endif()
 
+  if(CMAKE_SYSTEM_NAME STREQUAL FreeBSD)
+    foreach(L OpenGL OpenAL)
+    	set(PKG_CONFIG_LIBS_PRIVATE "${PKG_CONFIG_LIBS_PRIVATE} -l${L}")
```

Comment:
Please use the `${LIBS_PRIVATE}` populated in `utils.cmake`.

Please test that both
```
cc raylib.c `pkg-config --libs --cflags raylib`
```

and
```
cc raylib.c `pkg-config --static --libs --static --cflags raylib`
```

are able to compile a simple `raylib.c` file, e.g.:

```
#include <raylib.h>

int main(void)
{
    InitWindow(101, 20, "Ok?");
    SetTargetFPS(1);

    BeginDrawing();
    ClearBackground(RAYWHITE);
    DrawText("Seems so.", 2, 0, 20, DARKGREEN);
    EndDrawing();
    CloseWindow();
}
```

Reply:
```c
$ cc raylib.c `pkg-config --libs --cflags raylib`
Package raylib was not found in the pkg-config search path.
Perhaps you should add the directory containing `raylib.pc'
to the PKG_CONFIG_PATH environment variable
Package 'raylib', required by 'virtual:world', not found
/tmp/raylib-c1871e.o: In function `main':
raylib.c:(.text+0x1d): undefined reference to `InitWindow'
raylib.c:(.text+0x27): undefined reference to `SetTargetFPS'
raylib.c:(.text+0x2c): undefined reference to `BeginDrawing'
raylib.c:(.text+0x44): undefined reference to `ClearBackground'
raylib.c:(.text+0x73): undefined reference to `DrawText'
raylib.c:(.text+0x78): undefined reference to `EndDrawing'
raylib.c:(.text+0x7d): undefined reference to `CloseWindow'
cc: error: linker command failed with exit code 1 (use -v to see invocation)
$ ls
CMakeFiles		cmake_install.cmake	raylib.c		raymath.h
Makefile		libraylib.a		raylib.h		rlgl.h
audio.h			physac.h		raylib.pc
$ cc raylib.c `pkg-config --static --libs --static --cflags raylib`
Package raylib was not found in the pkg-config search path.
Perhaps you should add the directory containing `raylib.pc'
to the PKG_CONFIG_PATH environment variable
Package 'raylib', required by 'virtual:world', not found
/tmp/raylib-4ca849.o: In function `main':
raylib.c:(.text+0x1d): undefined reference to `InitWindow'
raylib.c:(.text+0x27): undefined reference to `SetTargetFPS'
raylib.c:(.text+0x2c): undefined reference to `BeginDrawing'
raylib.c:(.text+0x44): undefined reference to `ClearBackground'
raylib.c:(.text+0x73): undefined reference to `DrawText'
raylib.c:(.text+0x78): undefined reference to `EndDrawing'
raylib.c:(.text+0x7d): undefined reference to `CloseWindow'
cc: error: linker command failed with exit code 1 (use -v to see invocation)
```